### DISCUSSION TITLE
Comment flasher la bête ?

### DISCUSSION THREAD

**Discussion by faxm0dem** [multiplie-fr/minimit-beta#54] [Q&A] [answered]
Et question subsidiaire : comment ouvrir le boitier sans l'abîmer (j'imagine qu'on ne peut pas le flasher via le DIN ni même OTA ?)

**Comment by multiplie-fr** [comment]
Le boîtier s'ouvre en deux. Il suffit de le faire glisser de haut en bas.

**Comment by multiplie-fr** [comment, marked as answer]
Et pour flasher la bête il faut utiliser un programmeur comme le : https://docs.espressif.com/projects/esp-iot-solution/en/latest/hw-reference/ESP-Prog_guide.html

Pour le cablage il faut se référer au schéma de la bête :-) (https://github.com/multiplie-fr/minimit/tree/main/electronique)

**Comment by valguenichon** [comment]
Bonjour l'équipe,
Je m'arrache les cheveux sur le flashage.
J'ai voulu ajouter un service pour m'amuser.
Et je galère pour faire reconnaître les partitions dans Arduino IDE avec ESP32 Wrover Kit. En fait ça ne liste pas les partitions et pourtant j'ai bien un fichier partitions.csv à l'emplacement de mon sketch.
Résultat : le minimit n'est plus utilisable, le flashage s'est pourtant bien déroulé et j'ai un écran noir.
J'ai un doute sur le fait que ce soit moi vu que je n'ai rajouté qu'un service et je n'ai rien modifié au coeur du truc.
Je penche plutôt pour une config pas bonne de l'IDE.
J'ai besoin d'aide, en mode noob car je suis très très débutant.
Merci.

**Comment by faxm0dem** [comment]
Voici la réponse d' @omevel je cite : 

```
Alors déjà pour tout remettre en place tu peux utiliser les binaires ici
<https://www.multiplie.fr/stuff/pack-bin.zip> et reflasher le Minimit avec
(je te laisse modifier le /dev/....) :

"esptool" --chip esp32 --port "/dev/cu.usbserial-11201" --baud 921600
 --before default_reset --after hard_reset write_flash -e -z --flash_mode
dio --flash_freq 80m --flash_size 16MB 0x1000
"sketch_minimit.ino.bootloader.bin" 0x8000
"sketch_minimit.ino.partitions.bin" 0xe000 "boot_app0.bin" 0x10000
"sketch_minimit.ino.bin"

Dis moi si ça refonctionne déjà !

Olivier
```
